### DISCUSSION TITLE
menyoo 2.0

### DISCUSSION THREAD

**Discussion by gardien38** [MAFINS/MenyooSP#696] [General]
Good evening
I would like to write to you because the menyoo 2.0 menu mod no longer works recently an update of gta5 was made and nothing has happened since then when I launch the game with the menu mod in the files the game launches but refuses to start. display I tried to remove the menu mode is to launch the game the update is done is the game is launched but when I restart the game with the menu mode files the game updates again but still the same yuck it launches but doesn't work 
Do you have a solution?

**Comment by AmgedOsman** [comment]
Same here!

**Reply by gardien38** [reply]
it's the imput8.dll that doesn't work
